pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel"]
```

Comment:
What is wheel required for?

Reply:
`pyproject.toml` is defined in PEP 518. PEP 518 requires both setuptools and wheel; these requirements are further lifted by PEP 517, so that you can use alternative backend such as flit, but PEP 517 is newer (as in accepted later), and I believe pip only supports PEP 518 but not PEP 517 at the moment.

The reason I added a `pyproject.toml` was to ensure wheel as a build dependency. Without wheel, you can’t  run `setup.py bdist_wheel`, and `pip install .` uses `setup.py install` (egg-based, rather than wheel-based`, and on Windows this causes Unicode command line arts to degrade to CP1252 (setuptools issue I linked above). In practice though, pip at the moment still requires wheel to be installed beforehand, despite installing wheel in the build environment, so this pyproject.toml doesn’t accomplish much in the current state of python packaging.

In short, the `pyproject.toml` is not necessary, but it could be future-proof.

Reply:
Cool! Thanks!